packages/xo-web/src/xo-app/settings/plugins/index.js

```diff
@@ -246,6 +246,7 @@ class Plugin extends Component {
             <div className='form-group pull-right'>
               <ActionButton
                 btnStyle='primary'
+                disabled={!loaded}
```

Comment:
How will users know the button is disabled because the plugin is not loaded?

Reply:
I added a tooltip.